### PR TITLE
LPS-184638 Add a checkbox to set/unset the propagation of permissions when creating a new folder

### PR DESCRIPTION
**Pull request by ces-totrinh**
Hi @locpham97, @ces-quanhuynh
Please help review this PR and leave your comments. Thanks.
# Motivation
Ticket: [LPS-184638](https://issues.liferay.com/browse/LPS-184638)
Add a checkbox to the folder edit form when a user creates a new folder to set/unset the propagation of permissions
# Proposed Solution
There's a permission section in the folder edit form using `liferay-ui:input-permissions` tag lib. We custom this tag lib `liferay-ui:input-permissions` by adding a new attribute `permissionPropagationCheckboxLabel` to this tag lib attribute.

Changing the property from `documentMedia` to `permissionPropagationCheckboxLabel` can make the attribute no longer so specific and can be reused for other portlets, not just for the DM portlet. Checkbox label can also be adjusted differently between portlets.
# Screenshots
image

Thanks!

### CODE REVIEW

File: util-taglib/src/META-INF/liferay-ui.tld [hunk 1]
```diff
@@ -1305,12 +1299,18 @@
...
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
 		<attribute>
 			<name>reverse</name>
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 			<type>boolean</type>
 		</attribute>
+
```
Comment (locpham97) [inline util-taglib/src/META-INF/liferay-ui.tld:1313]: Try to SF this file

**Review by ces-locpham — APPROVED** [review]

**Review by ces-quanhuynh — APPROVED** [review]
LGTM!

**Comment by ces-totrinh** [comment]
@hhtank, this PR is passed our review. Please have a look. Thanks.

**Comment by hhtank** [comment]
Hi team, this PR looks good to me. Feel free to proceed next steps. Thanks,

@tototrinh

**Pull request closed without merging**